src/test/scala/it/pagopa/interop/purposemanagement/provider/PurposeVersionStateChangeSpec.scala

```diff
@@ -73,25 +32,24 @@ class PurposeVersionStateChangeSpec extends BaseIntegrationSpec {
         description = "Purpose description",
         riskAnalysisForm = Some(riskAnalysisFormSeed),
         isFreeOfCharge = false,
-        freeOfChargeReason = None
+        freeOfChargeReason = None,
+        dailyCalls = 100
       )
-      val versionSeed = PurposeVersionSeed(riskAnalysis = None, dailyCalls = 100)
 
-      val response: Future[(PurposeVersion, PurposeVersion)] =
+      val response: Future[PurposeVersion] =
         for {
-          _       <- createPurpose(purposeId, purposeSeed)
-          version <- createPurposeVersion(purposeId, versionId, versionSeed)
-          result  <- activateVersion(purposeId, versionId, ChangedBy.CONSUMER, Some(riskAnalysisDoc))
-        } yield (version, result)
+          _      <- createPurpose(purposeId, versionId, purposeSeed)
+          result <- activateVersion(purposeId, versionId, ChangedBy.CONSUMER, Some(riskAnalysisDoc))
+        } yield result
 
-      val (version, result) = response.futureValue
```

Comment:
this is testing that on activation, those (and only) specific fields are updated.
This is why it was returning the created version and the activated version.
With this update, if a new field that we do not expect is changed the test will succeed.

This should be rollbacked